controllers/packagebundlecontroller_controller.go

```diff
@@ -113,20 +113,31 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 		return withoutRequeue(result), nil
 	}
 
+	changed := false
 	if pbc.Status.State != api.BundleControllerStateActive {
 		pbc.Status.State = api.BundleControllerStateActive
-		err = r.Client.Status().Update(ctx, pbc, &client.UpdateOptions{})
-		if err != nil {
-			return result, fmt.Errorf("updating status to active: %s", err)
-		}
+		changed = true
 	}
 
 	latestBundle, err := r.bundleManager.LatestBundle(ctx, pbc.Spec.Source.BaseRef())
 	if err != nil {
 		r.Log.Error(err, "Unable to get latest bundle")
+		if pbc.Status.State != api.BundleControllerStateDisconnected {
```

Comment:
This is related to #109 and it was a request during our high level design review. If for whatever reason we cannot download new bundles, we should notify the customer that we are in an error state. They can tell they are in an error state using kubectl instead of looking at logs

Reply:
Hmm, thanks. Shoot, I see now that you linked that issue in your PR description, and I just missed it cuz of the nice table there.

Reply:
I might need a little time to get used to disconnected as the term. It doesn't feel right to me, but I can't think of anything better. :)